### ISSUE TITLE
[hax] Title missing from left-hand pane

### ISSUE THREAD

**Issue by JackWillDavis** [elmsln/issues#526]
In the HAX left-hand pane, there is no title showing when the inputMethod is set to `code-editor` so it can look a little mixed up with the other options:

![image](https://user-images.githubusercontent.com/2301874/107542977-a9b47180-6bc0-11eb-9a2d-fa0ddcb12066.png)

Might also be good for titles to be shown in bold/larger to separate out each option too, as below:

![image](https://user-images.githubusercontent.com/2301874/107543429-234c5f80-6bc1-11eb-9d7f-346f3f2cd56e.png)

**Comment by btopro** [comment]
@nikkimk I believe this is related to the way the simple-fields schema is interpreted. Does this seem reasonable / an a11y enhancement to make sure there's a title displayed on code editing as it is for everything else? activity-box is located under `course-design/lib` as it's a newer element
